metaflow/metaflow_config.py

```diff
@@ -154,6 +154,7 @@ def from_conf(name, default=None):
 ARGO_RUN_URL_PREFIX = from_conf("ARGO_RUN_URL_PREFIX", "")
 METAFLOW_RUN_URL_PREFIX = from_conf("METAFLOW_RUN_URL_PREFIX", "")
 KFP_MAX_PARALLELISM = int(from_conf("KFP_MAX_PARALLELISM", 10))
```

Comment:
I know you mentioned in the meeting but feel we should not have `KFP` for these values.